maps/src/test/java/org/odk/collect/maps/layers/OfflineMapLayersPickerTest.kt

```diff
@@ -108,11 +127,11 @@ class OfflineMapLayersPickerTest {
 
     @Test
     fun `clicking the 'save' button saves null when 'None' option is checked`() {
-        whenever(referenceLayerRepository.getAll()).thenReturn(listOf(
-            ReferenceLayer("1", TempFiles.createTempFile(), "layer1")
-        ))
+        whenever(referenceLayerRepository.getAll()).thenReturn(
```

Comment:
Probably worth creating an `InMem...` version of `ReferenceLayerRepository` to make these tests a little easier to read.